pkg/eden/vbox.go

```diff
@@ -0,0 +1,251 @@
+		}
+
+		commandArgsString = fmt.Sprintf("storagectl %s --name \"SATA\" --add sata --bootable on --hostiocache on", vmName)
+		if err = utils.RunCommandWithLogAndWait("VBoxManage", defaults.DefaultLogLevelToPrint, strings.Fields(commandArgsString)...); err != nil {
+			log.Fatalf("VBoxManage error for command %s %s", commandArgsString, err)
+		}
+
+		commandArgsString = fmt.Sprintf("storageattach %s  --storagectl \"SATA\" --port 0 --device 0 --type hdd --medium %s", vmName, eveImageFile)
+		if err = utils.RunCommandWithLogAndWait("VBoxManage", defaults.DefaultLogLevelToPrint, strings.Fields(commandArgsString)...); err != nil {
+			log.Fatalf("VBoxManage error for command %s %s", commandArgsString, err)
+		}
+
+		commandArgsString = fmt.Sprintf("natnetwork add --netname %s --network %s --enable --dhcp on",
+										"natnet1", defaults.DefaultVBoxSubnet)
+		if err = utils.RunCommandWithLogAndWait("VBoxManage", defaults.DefaultLogLevelToPrint, strings.Fields(commandArgsString)...); err != nil {
+			log.Fatalf("VBoxManage error for command %s %s", commandArgsString, err)
+		}
+		commandArgsString = fmt.Sprintf("natnetwork start --netname %s", "natnet1")
+		if err = utils.RunCommandWithLogAndWait("VBoxManage", defaults.DefaultLogLevelToPrint, strings.Fields(commandArgsString)...); err != nil {
+			log.Errorf("VBoxManage error for command %s %s", commandArgsString, err)
+		}
+
+		commandArgsString = fmt.Sprintf("modifyvm %s --nic1 natnetwork --nat-network1 %s --cableconnected1 on",
+										vmName, "natnet1")
+		if err = utils.RunCommandWithLogAndWait("VBoxManage", defaults.DefaultLogLevelToPrint, strings.Fields(commandArgsString)...); err != nil {
+			log.Fatalf("VBoxManage error for command %s %s", commandArgsString, err)
+		}
+
+		eth0IP := ipMap["eth0"]
+		for k, v := range hostFwd {
+			commandArgsString = fmt.Sprintf("natnetwork  modify --netname %s --port-forward-4 :tcp:[]:%s:[%s]:%s",
+											"natnet1", k, eth0IP.String(), v)
+			if err = utils.RunCommandWithLogAndWait("VBoxManage", defaults.DefaultLogLevelToPrint, strings.Fields(commandArgsString)...); err != nil {
+				log.Fatalf("VBoxManage error for command %s %s", commandArgsString, err)
+			}
+		}
+
+		commandArgsString = fmt.Sprintf("modifyvm %s  --nic2 natnetwork --nat-network2 %s --cableconnected2 on",
+										vmName, "natnet1")
+		if err = utils.RunCommandWithLogAndWait("VBoxManage", defaults.DefaultLogLevelToPrint, strings.Fields(commandArgsString)...); err != nil {
+			log.Fatalf("VBoxManage error for command %s %s", commandArgsString, err)
+		}
+		eth1IP := ipMap["eth1"]
+		for k, v := range hostFwd {
+			hostPort, err := strconv.Atoi(k)
+			if err != nil {
+				log.Errorf("Parsing %s to Integer value failed", k)
+				break
+			}
+			hostPort += 10
+			guestPort, err := strconv.Atoi(v)
+			if err != nil {
+				log.Errorf("Parsing %s to Integer value failed", v)
+				break
+			}
+			guestPort += 10
```

Comment:
We have default for this. And the same point for another parts.

Reply:
The discussion above applies to this also.